client/src/containers/routing/index.tsx

```diff
@@ -1,13 +1,45 @@
 import React from 'react';
 import { Switch } from 'react-router-dom';
-import Example from 'containers/example';
-import { PublicRoute } from 'components';
+import { TaskInstructions } from 'components';
 
-interface IRoutingProps { }
+interface IRoutingProps {}
+
+const html = (
+	<>
+		<p>
+			Write a function that takes in a string of one or more words, and returns the same string, but with all five
+			or more letter words reversed.
+		</p>
+		<ul>
+			<li>Strings passed in will consist of only letters and spaces.</li>
+			<li>Spaces will be included only when more than one word is present.</li>
+		</ul>
+		<h3>Examples:</h3>
+		<div className="example">
+			<p>{`spinWords("Hey fellow warriors") => "Hey wollef sroirraw"`}</p>
+			<p>{`spinWords("This is a test") => "This is a test"`}</p>
+			<p>{`spinWords("This is another test") => "This is rehtona test"`}</p>
+		</div>
+		<div className="tags">
+			<a>Algorithms</a>
+			<a>Strings</a>
+			<a>Data Types</a>
+			<a>Formatting</a>
+			<a>Logic</a>
+		</div>
+	</>
+);
 
 const Routing: React.FC<IRoutingProps> = () => (
 	<Switch>
-		<PublicRoute exact restricted={false} path="/" component={Example} />
+		{/* <PublicRoute exact restricted={false} path="/" component={} /> */}
+		<TaskInstructions
+			title="Stop gninnipS My sdroW!"
+			html={html}
+			favorites={500}
+			rating={88}
+			createdBy="Emerson Saris"
+		/>
```

Comment:
You should not mock data like this. Let's create a container and mock data from container. Container should return component or several components.

```javascript
const mockData = {
   rating: 88,
   favorites: 500,
   /* .... */
}
```

See example how it should be from /users/:name route container.

You need to create route for your page too. Let it be /task/:id